### PR TITLE
update march+april schedule

### PR DESCRIPTION
**Pull request by afeld**
/cc @cuny-nytech/students

### CODE REVIEW

File: schedule.md [hunk 1]
```diff
@@ -154,29 +154,87 @@
...
+
 ## Mar 16
 
+* Present interview results
+* In-person chat w/ Ricky
+* GUEST: Jeremy Canfield from 18F, on User-Centered Design
+
+## Mar 21, 11am-4pm
+
+Visit CoderDojo event at SquareSpace.
```
Comment (afeld) [inline schedule.md]: @cuny-nytech/students Let me know if you can't do this!

File: schedule.md [hunk 2]
```diff
@@ -154,29 +154,87 @@
...
+
+* Observe event setup procedures
+* Observe registration procedures
+    * Try to talk to some first-time attendees/parents, and those who have attended before
+    * Take note of issues/bottlenecks/confusion
+
+## Mar 23
+
+* Aidan out of town
 * Professional presence, interview prep - Peter
```
Comment (afeld) [inline schedule.md]: @PeterBell Are you free on this day? I'll be out of town, so would be convenient if you could do your talk. Or @dbrizan?

Comment (afeld) [inline schedule.md]: @PeterBell @dbrizan Bump!

**Comment by PeterBell** [comment]
@afeld Dang - I am out of town in mexico :( @dbrizan ?

File: schedule.md (hunk 2 above)
Comment (afeld) [inline schedule.md]: @dbrizan FWIW the topic doesn't have to be "professional presence"...totally open to other ideas. You could even talk a bit about your research if you like. "Worst" case, we can leave the evening as an open work session for the team.

Comment (dbrizan) [inline schedule.md]: Sorry for the delay. I believe I'm free. Will sync with AF privately.

**Pull request merged**